### ISSUE TITLE
Non-white background can be misleading

### ISSUE THREAD

**Issue by joedmck** [jbt/markdown-editor#93]
It would be more helpful if the background was white like that of the github md viewer. This was annoying when I pasted an image with a white background, and until I checked that the background on the github site was white, thought it was going to look quite bad.

**Comment by aero31aero** [comment]
I think I should revert it then. My reasoning was that it was slightly easier on the eyes but staying close to GitHub's implementation is the main objective.

**Comment by joedmck** [comment]
Yeah it would be good @aero31aero if all pictures embedded had transparent backgrounds, but when I embed an image with a white background I always doubt myself and go and check that it'd work fine on GitHub itself. 

I actually cloned this repo onto my own web server and changed the background to white. Only opened the issue because it could mislead others.

**Comment by aero31aero** [comment]
Merged and deployed!
